themes/he/layouts/_default/voice_bot.html

```diff
@@ -65,6 +65,7 @@ <h1 class="h1">{{ . | markdownify }}</h1>
         data-target="#myModal"
         type="submit"
         class="btn btn-primary mt-15"
+        style="-webkit-appearance: none; -moz-appearance: none; appearance: none"
```

Comment:
@arunkumar295 check this in demo site and then confirm.

Reply:
checked on demo site button appears in correct shape and color.
![Screenshot 2023-01-03 at 5 51 50 PM](https://user-images.githubusercontent.com/115898232/210358103-b2eedfb6-1576-486e-a63b-54682eb8c2a5.png)